floss/main.py

```diff
@@ -523,26 +523,24 @@ def main(argv=None) -> int:
     interim = time0
     sample_size = os.path.getsize(sample)
 
+    with open(sample, "rb") as f:
+        with contextlib.closing(mmap.mmap(f.fileno(), 0, access=mmap.ACCESS_READ)) as buf:
```

Comment:
im not sure this is the right set of flags on both Linux and Windows. my best guess for this in the past is here: https://github.com/mandiant/flare-floss/blob/25b6f85dd45f7a47bcc90f3c94b328fe2477f0b0/floss/qs/main.py#L610-L620

Reply:
this is what FLOSS used before, but would be good if @Arker123 can confirm here this actually works as expected :)